ci/promote.py

```diff
@@ -35,7 +35,7 @@
     },
     'root': {
         'level': logging.DEBUG,
-        'handlers': ['console',],
+        'handlers': ['console', ],
```

Comment:
I suppose this is from auto-code-formatter?

Reply:
yes... autopep8